src/Kunstmaan/GeneratorBundle/Command/InstallCommand.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Kunstmaan\GeneratorBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Output\OutputInterface;
+use Sensio\Bundle\GeneratorBundle\Command\Validators;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Process\Exception\RuntimeException;
+use Sensio\Bundle\GeneratorBundle\Command\Helper\QuestionHelper;
+use Symfony\Component\Debug\Exception\ContextErrorException;
+
+/**
+ * Kunstmaan installer
+ */
+class InstallCommand extends Command
+{
+    /**
+     * @var int
+     */
+    private $commandSteps = 0;
+
+    /** @var string */
+    private $rootDir;
+
+    /**
+     * @param string $rootDir
+     */
+    public function __construct(string $rootDir)
```

Comment:
When the service parameter is changed from root_dir to project_dir, this parameter and property should be renamed